randovania/games/fusion/assets/hash_words.json

```diff
@@ -0,0 +1,113 @@
+[
+    "Animals",
+    "Arachnus",
+    "B.O.X.",
+    "Bull",
+    "Choot",
+    "Core-X",
+    "Dachora",
+    "Dessgeega",
+    "Etecoon",
+    "Evir",
+    "Fune",
+    "Gadora",
+    "Geemer",
+    "Genesis",
+    "Geron",
+    "Gerubus",
+    "Geruta",
+    "Halzyn",
+    "Hornoad",
+    "Kago",
+    "Kihunter",
+    "Metroid",
+    "Moto",
+    "Namihe",
+    "Nettori",
+    "Nightmare",
+    "Omega",
+    "Owtch",
+    "Parasite",
+    "Pirate",
+    "Powamp",
+    "Puyo",
+    "Reo",
+    "Ridley",
+    "Ripper",
+    "Robot",
+    "SA-X",
+    "Scientist",
+    "Sciser",
+    "Serris",
+    "Sidehopper",
+    "Skultera",
+    "Waver",
+    "Yakuza",
+    "Yameba",
+    "Zazabi",
+    "Zebesian",
+    "Zeela",
+    "Zero",
+    "Zombie",
+    "Zoro",
+    "Zozoro",
+    "SRX",
+    "TRO",
+    "PYR",
+    "AQA",
+    "ARC",
+    "NOC",
+    "Access",
+    "Acid",
+    "Atmospheric",
+    "Authorization",
+    "B.S.L",
+    "Bay",
+    "Biological",
+    "Boiler",
+    "Containment",
+    "Control",
+    "Cooling",
+    "Data",
+    "Deck",
+    "Diffusion",
+    "Docking",
+    "Download",
+    "Elevator",
+    "Emergency",
+    "Federation",
+    "Fusion",
+    "Galactic",
+    "Habitation",
+    "Hangar",
+    "Hatch",
+    "Laboratory",
+    "Main",
+    "Meltdown",
+    "Navigation",
+    "Operations",
+    "Pump",
+    "Quarantine",
+    "Reactor",
+    "Recharge",
+    "Restricted",
+    "Sector",
+    "Security",
+    "Silo",
+    "Space",
+    "Stabilizer",
+    "Station",
+    "Sub-Zero",
+    "Tank",
+    "Unit",
+    "Vines",
+    "Zone",
+    "Grey",
+    "Blue",
+    "Green",
+    "Yellow",
+    "Red",
+    "Adam",
+    "Bob",
```

Comment:
Do we want bob on the hashlist? to my knowledge its a community given name and sounds a bit weird to have here.